build-tools/build-indexes

```diff
@@ -301,6 +302,17 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 				if (template.tier === 'CAP' || template.tier === 'CAP NFE' || template.tier === 'CAP LC') {
 					return template.tier;
 				}
+				if (isLetsGo) {
+					let baseTemplate = Tools.mod(gen).getTemplate(template.baseSpecies);
+					if ((baseTemplate.num > 151 || baseTemplate.num < 1) && baseTemplate.num !== 808 && baseTemplate.num !== 809) {
+						if (!['Alola', 'Mega', 'Mega-X', 'Mega-Y', 'Starter'].includes(template.forme)) return 'Illegal';
```

Comment:
err, this line does nothing

Reply:
I'm pretty sure you're making the same mistake you made server-side.

Reply:
https://github.com/Zarel/Pokemon-Showdown/commit/ed54a6eb4db5d5271174cc42537432bb0bf1e602

You understand how this change made the logic a lot clearer, and makes it a lot harder to make mistakes, right?

Reply:
Can you do the same kind of change here?

Reply:
done